mappings/net/minecraft/util/math/Quaternion.mapping

```diff
@@ -1,10 +1,14 @@
 CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
 	FIELD field_21493 IDENTITY Lnet/minecraft/class_1158;
+	FIELD field_21582 x F
+	FIELD field_21583 y F
+	FIELD field_21584 z F
+	FIELD field_21585 angle F
 	METHOD <init> (FFFF)V
-		ARG 1 b
-		ARG 2 c
-		ARG 3 d
-		ARG 4 a
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 angle
```

Comment:
this is like `a + bi + cj + dk` (https://en.wikipedia.org/wiki/Quaternion)
Also the constructor of a vector and an angle has a few transformations beyond as well.

Reply:
Actually, I think something went wrong with the updating. I see this constructor _was_ mapped, but the names are now associated with the wrong method.


```
    // the one I mapped
    public Quaternion(float x, float y, float z, float angle) {
        this.x = x;
        this.y = y;
        this.z = z;
        this.angle = angle;
    }

    @Environment(EnvType.CLIENT)
    public Quaternion(float x, float y, float z, boolean degrees) {
        if (degrees) {
            x *= 0.017453292f;
            y *= 0.017453292f;
            z *= 0.017453292f;
        }
        float float6 = sin(0.5f * x);
        float float7 = cos(0.5f * x);
        float float8 = sin(0.5f * y);
        float float9 = cos(0.5f * y);
        float float10 = sin(0.5f * z);
        float float11 = cos(0.5f * z);
        this.x = float6 * float9 * float11 + float7 * float8 * float10;
        this.y = float7 * float8 * float11 - float6 * float9 * float10;
        this.z = float6 * float8 * float11 + float7 * float9 * float10;
        this.angle = float7 * float9 * float11 - float6 * float8 * float10;
    }
```

Reply:
Actuall, no. Nevermind I'm being silly. Still, this specific one is for a basic constructor with no transform.

Reply:
No, those are not xyz and angle...

Reply:
I don't really know what else you can call them then.

Reply:
Please restore the old b,c,d,a names. They are accepted as seen on wikipedia.